dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Basics/CombiTimeTable.mo

```diff
@@ -0,0 +1,41 @@
+within Dynawo.Electrical.Controls.Basics;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model CombiTimeTable "Table look-up with respect to time and linear/periodic extrapolation methods (data from a two-column table in a .txt file) with one output"
+  import Modelica;
+  import Dynawo;
+
+  parameter Modelica.Blocks.Types.Extrapolation Extrapolation
+    "Extrapolation of data outside the definition range";
+  parameter String FileName = "NoName" "File where matrix is stored"
+    annotation(Dialog(
+      enable=TableOnFile,
+      loadSelector(filter="Text files (*.txt);;MATLAB MAT-files (*.mat)",
+          caption="Open file in which table is present")));
+  parameter Modelica.Blocks.Types.Smoothness Smoothness
```

Comment:
Is it useful ? Does it work fine in dynawo ?

Reply:
Thus, we may approve of this component for ConstantSegments only while opening an issue about the linear interpolation under Dynawo.

Reply:
Ok, could we check with different solver parameters ? Thanks